client/src/App.jsx

```diff
@@ -1,14 +1,21 @@
-import { CssBaseline, ThemeProvider } from '@mui/material';
 import React from 'react';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import { CssBaseline, ThemeProvider } from '@mui/material';
 import Router from './pages/routing';
 import { theme } from 'theme';
 
+const queryClient = new QueryClient();
+
 const App = () => {
   return (
-    <ThemeProvider theme={theme}>
-      <CssBaseline />
-      <Router />
-    </ThemeProvider>
+    <>
+      <ThemeProvider theme={theme}>
+        <CssBaseline />
+        <QueryClientProvider client={queryClient}>
+          <Router />
+        </QueryClientProvider>
+      </ThemeProvider>
+    </>
```

Comment:
Fragment nie jest potrzebny w tym miejscu.

```suggestion
      <ThemeProvider theme={theme}>
        <CssBaseline />
        <QueryClientProvider client={queryClient}>
          <Router />
        </QueryClientProvider>
      </ThemeProvider>
```